package.json

```diff
@@ -79,6 +79,7 @@
     "nodemon": "^1.17.2",
     "require-dir": "^1.0.0",
     "ts-node": "^6.0.2",
+    "ts-protoc-gen": "^0.6.0",
```

Comment:
I think we should add the script that is using that lib into our project, to be executable through `gulp` or `npm scripts`, so we'll know how to regenerate `lndrpc_pb.d.ts` after we're upgrading `lndrpc.proto`.
At the moment, I don't see this lib used through our project, so I assume you just used it to generate `lndrpc_pb.d.ts`, which in this case it doesn't need to be saved in our `package.json`, and can be used as a global npm package.
If that's the case, still, instead of removing it I think we should add the generating script to our automation.

Reply:
Agree with @moshababo it would be a good idea to include it in npm scripts, maybe compile itself?

Reply:
Agreed, having a script was always my intention, I just generated this on windows and need to set this up on linux to get the command right. I'll work on it.